code/game/gamemodes/wizard/wizard.dm

```diff
@@ -248,6 +248,20 @@
 		qdel(spell_to_remove)
 		mind.spell_list -= spell_to_remove
 
+/datum/mind/proc/add_spell(var/obj/effect/proc_holder/spell/spell) //To add a specific spell to a mind

```

Comment:
There already is AddSpell - handling action buttons etc for both minds and mobs.